examples/supremacy.py

```diff
@@ -17,10 +17,10 @@
 from typing import Callable, Iterable
 
 import cirq
-from cirq import devices, google
+import cirq.google as cg
```

Comment:
Why import as cg?  Why not just as google?  Seems like it makes the code less readable.

Reply:
It's for the same reason people import numpy as np.